requirements.txt

```diff
@@ -18,12 +18,15 @@ mypy
 trustme
 cryptography
 coverage
-httpx==0.16.*
+httpx>=0.18.2
 pytest-asyncio==0.14.*
 async_generator; python_version < '3.7'
 
 
 # Documentation
-mkdocs
+# from git due to https://github.com/mkdocs/mkdocs/pull/2209#issuecomment-855703485
+https://github.com/mkdocs/mkdocs/archive/b221df94fa7f29a2e63edfaeda56e3285b183f91.tar.gz#egg=mkdocs
 mkdocs-material
 
+# py3.10 workarounds
+https://github.com/aaugustin/websockets/archive/0bfa9f2ea86dbc12f24e4a2756cb339efafe76d8.tar.gz#egg=websockets; python_version > '3.6'
```

Comment:
on hold until this is released, won't be able to upload to pypi anyway if merged

Reply:
```suggestion
https://github.com/aaugustin/websockets/archive/d5cf1efb737a943583b1e5b4ceca5376bdc3995f.tar.gz#egg=websockets; python_version > '3.6'
```

Reply:
@euri10 I've changed this to use `; python_verion >= '3.10'` so we can upload this to pypi now